tests/publiccloud/migration.pm

```diff
@@ -36,7 +36,8 @@ sub run {
 
     record_info('installs SLE15-Migration and suse-migration-sle15-activation');
 
-    $instance->run_ssh_command(cmd => "sudo zypper -n in SLES15-Migration suse-migration-sle15-activation", timeout => 300);
+    $instance->run_ssh_command(cmd => "sudo zypper -n in SLES15-Migration suse-migration-sle15-activation", timeout => 360);
```

Comment:
I fixed this already yesterday in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15805. Are those additional 60 seconds really necessary?

Reply:
Earlier it was 90 , I had issues with Azure image. It used to timeout https://openqa.suse.de/tests/9802122#step/migration/168

Let me test for couple of days. If 300 works, I can leave it as it's

Reply:
Also I would like to understand, What was your intention to increase the timeout in migration script ??

Reply:
I'm on QA-C job review duty this week, so yes, I'm monitoring all jobs that are in our production pipeline. I saw the migration tests failing, filed https://progress.opensuse.org/issues/119602 for it and provided a quickfix in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15805.